cytominer_database/ingest.py

```diff
@@ -103,14 +103,38 @@ def into(input, output, name, identifier, skip_table_prefix=False):
             con = engine.connect()
 
             df = pd.read_csv(source, index_col=0)
-            df.to_sql(name=name, con=con, if_exists="append")
+    """
+        
+    with warnings.catch_warnings():
+        # Suppress the following warning on Python 3:
+        #
+        #   /usr/local/lib/python3.6/site-packages/odo/utils.py:128: DeprecationWarning: inspect.getargspec() is
+        #     deprecated, use inspect.signature() or inspect.getfullargspec()
+        warnings.simplefilter("ignore", category=DeprecationWarning)
+        engine = create_engine(output)
+        con = engine.connect()
+
+        df = pd.read_csv(input)
+        #print("In into(): df['Index']") # no index prepended yet
+        #print(df['Index'])
+        # add "name"prefix to column headers
+        if not skip_table_prefix:
+            no_prefix = ["ImageNumber", "ObjectNumber"]  # exception columns
+            df.columns = [
+                i if i in no_prefix else name + "_" + i for i in df.columns
```

Comment:
```suggestion
                i if i in no_prefix else f"{name}_{i}" for i in df.columns
```

[f strings](https://realpython.com/python-f-strings/) are legit

Reply:
any reason to not accept this "suggestion"? Github makes these "suggestions" easy to manage. Let's hop on a call next week to discuss :)

Reply:
@diskontinuum - let's discuss how to "accept" a commit via the github gui. it really is quite nice! (and is very close to zero effort)

Reply:
@gwaygenomics : I kept ingest.py as an unmodified copy of the original :) , hence I did not introduce any changes. 
The modified code is in ingest_incl_parquet.py, where I introduced the fstring for this list comprehension :) . 
BTW, this suggestion failed at the Travis CI - I then rewrote the line without ist comprehensions (or fstrings).

Reply:
* Now I introduced this change to the original as well.